cpu/xive.py

```diff
@@ -72,15 +72,25 @@ def test_storeEOI(self):
                           "missing %s / %s" % (self.hw, self.intr))
             if is_dir:
                 # If xive is a directory then read from store-eoi file
-                xive_path = "/sys/kernel/debug/powerpc/xive/store-eoi"
-
-            flags = genio.read_file(xive_path)
-            if is_dir:
-                # store-eoi value can be enabled or disable via kernel command
-                # line or using sysfs debug directory.
-                match = re.search("Y", flags) or re.search("N", flags)
+                # store_eoi_file = os.path.join(xive_path, "store-eoi")
```

Comment:
@rohan-r-deshpande is this line required?

Reply:
@rohan-r-deshpande can you please address this review comment

Reply:
@Naresh-ibm @PraveenPenguin Done. Please check!